packages/web-components/package.json

```diff
@@ -84,12 +84,12 @@
     "rollup-plugin-transform-tagged-template": "^0.0.3",
     "rollup-plugin-typescript2": "^0.27.0",
     "source-map": "^0.7.3",
-    "source-map-loader": "^0.2.4",
-    "ts-loader": "^7.0.2",
+    "source-map-loader": "^2.0.0",
+    "ts-loader": "^8.0.14",
     "ts-node": "^8.0.0",
     "tsconfig-paths": "^3.9.0",
     "typescript": "^3.9.0",
-    "webpack": "^4.43.0"
+    "webpack": "^5.12.3"
```

Comment:
Now that you're doing this you can get rid of some stuff in the root package.json:
- `syncpack.versionGroups[0].dependencies` webpack
- `workspaces.nohoist` webpack entries